doc/tutorials/using_ubuntu_as_sos.rst

```diff
@@ -13,11 +13,11 @@ Install Ubuntu (natively)
 *************************
 
 Ubuntu 16.04.4 LTS was used throughout this document, other versions such as
-18.04 may work too.
+18.04 works too.
```

Comment:
If you know that 18.04 works, it would be better to change the documentation below to use this latest release, and mention here that the older 16.04.4 version works too, and use the current 18.04 release links instead of the "old-releases" links below.